modules/sbo-release-notes-1-1.adoc

```diff
@@ -0,0 +1,95 @@
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+[id="support-matrix-1-1_{context}"]
+== Support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Support matrix
+[cols="1,1",options="header"]
+|===
+| Feature | {servicebinding-title} 1.1
+| `binding.operators.coreos.com` API group
+| GA
+| `servicebinding.io` API group
+| TP
+|===
+
+[id="new-features-1-1_{context}"]
+== New features
+This section highlights what is new in {servicebinding-title} 1.1:
+
+* Service Binding Options
+** Workload resource mapping: Define exactly where binding data needs to be projected for the secondary workloads.
+** Bind new workloads using a label selector
+
+
+[id="fixed-issues-1-1_{context}"]
+== Fixed issues
+* Before this update, service bindings that used label selectors to pick up workloads did not project service binding data into the new workloads that matched the given label selectors. As a result, the Service Binding Operator could not periodically bind such new workloads. With this update, service bindings now project service binding data into the new workloads that match the given label selector. The Service Binding Operator now periodically attempts to find and bind such new workloads. link:https://issues.redhat.com/browse/APPSVC-1083[APPSVC-1083]
+
+
+[id="known-issues-1-1_{context}"]
+== Known issues
+* Currently, when you install the {servicebinding-title} in the single namespace installation mode, the absence of an appropriate namespace-scoped role-based access control (RBAC) rule prevents the successful binding of an application to a few known Operator-backed services that the {servicebinding-title} can automatically detect and bind to. In addition, the following error message is generated:
++
+.Example error message
+----
+`postgresclusters.postgres-operator.crunchydata.com "hippo" is forbidden:
+        User "system:serviceaccount:my-petclinic:service-binding-operator" cannot
+        get resource "postgresclusters" in API group "postgres-operator.crunchydata.com"
+        in the namespace "my-petclinic"`
+----
++
+Workaround 1: Install the {servicebinding-title} in the `all namespaces` installation mode. As a result, the appropriate cluster-scoped RBAC rule now exists and the binding succeeds.
++
+Workaround 2: If you cannot install the {servicebinding-title} in the `all namespaces` installation mode, install the following role binding into the namespace where the {servicebinding-title} is installed: 
```

Comment:
Similar questions about using the term "mode" here.

Reply:
Ignored :)